core/src/test/java/com/sequenceiq/cloudbreak/util/ParcelAccessibilityTest.java

```diff
@@ -26,29 +28,47 @@
 
 public class ParcelAccessibilityTest {
 
+    private static Map<String, Object> appYaml;
+
+    @BeforeAll
+    public static void setUp() throws IOException, URISyntaxException {
+        appYaml = readCoreAppYaml();
+    }
+
+    @Test
+    public void testParcelAccessibility() {
+        List<DefaultCDHInfo> defaultCDHInfo = getCsds(appYaml);
+        defaultCDHInfo.stream()
+                .filter(s -> !s.getParcels().isEmpty())
+                .flatMap(s -> s.getParcels().stream())
+                .flatMap(s -> s.getCsd().stream())
+                .forEach(this::validateUrl);
+    }
+
     @Test
-    public void testParcelAccessibility() throws URISyntaxException, IOException {
+    void validateParcelUrls() {
+        List<DefaultCDHInfo> defaultCDHInfo = getCsds(appYaml);
+        defaultCDHInfo.stream()
+                .filter(s -> !s.getRepo().getStack().isEmpty())
+                .flatMap(s -> s.getRepo().getStack().entrySet().stream())
+                .map(s -> s.getValue() + "manifest.json")
```

Comment:
Is the parcel url should be ended with `/`?

Reply:
So far all parcel URL's ended with a slash and from now it will be enforced.